lib/providers/services/firebase/push_notification_service.dart

```diff
@@ -13,7 +17,7 @@ class PushNotificationService {
   bool _initialized = false;
   String token;
 
-  Future initialise() async {
+  Future initialise(BuildContext context) async {
     if (!_initialized) {
       if (Platform.isIOS) {
         _firebaseMessaging.onIosSettingsRegistered.listen((data) {
```

Comment:
I am guessing initialization like this is not needed on Android? @gguijarro-c-chwy

Reply:
No, this is only for IOS